components/service/glean/src/main/java/mozilla/components/service/glean/CommonMetricData.kt

```diff
@@ -24,6 +24,8 @@ enum class Lifetime {
     User
 }
 
+private val logger = Logger("glean/CommonMetricData")
```

Comment:
Mh, I don't think we need this: `shouldRecord` already has a `logger` parameter that can be used for this.

Reply:
Thanks, I missed that.